sdk/storage/storage-file-share/src/generated/src/operations/share.ts

```diff
@@ -433,13 +430,12 @@ const createPermissionOperationSpec: coreHttp.OperationSpec = {
     Parameters.version
   ],
   requestBody: {
-    parameterPath: "permission",
+    parameterPath: "sharePermission",
     mapper: {
-      ...Mappers.Permission,
+      ...Mappers.SharePermission,
       required: true
     }
   },
-  contentType: "application/xml; charset=utf-8",
```

Comment:
![image](https://user-images.githubusercontent.com/10452642/68731155-f6576a00-0583-11ea-980c-05f690a3436e.png)

One live test failed because of the update in content-type `application/json`(unified swagger).
Trying to fix it with a swagger transform.